src/components/Logo/Logo.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
Again, no need for it.
Check if your VS-Code is showing you un-used imports so you will be able to identify them easily and delete them.